src/kafka-topics/list/topics-list.controller.js

```diff
@@ -51,7 +51,9 @@ angularAPP.controller('KafkaTopicsListCtrl', function ($scope, $rootScope, $rout
     }, function (update) {
       $log.info('Got notification: ' + update);
     });
-
+  };
+  $interval(start, 5000);
```

Comment:
I think we'll figure a better way - instead of re-pulling every 5sec
Other changes proposed - have already been incorporated
Still reviewing the scope of this pull - request - not sure we'll adopt 100% of it